debuglog-plugin/src/test/kotlin/com/bnorm/debug/log/IrPluginTest.kt

```diff
@@ -56,7 +56,7 @@ fun doSomething() {
     val result = compile(sourceFile = main, DebugLogComponentRegistrar(true))
     assertEquals(KotlinCompilation.ExitCode.OK, result.exitCode)
 
-    val out = invokeMain(result, "MainKt").trim().split("\n")
+    val out = invokeMain(result, "MainKt").trim().split(Regex("""[\r\n]+"""))
```

Comment:
Not really something to stress about, but I don't like that this could match a single carriage return character. Should probably be more like this?
```suggestion
    val out = invokeMain(result, "MainKt").trim().split("""\r?\n+""".toRegex())
```

(I also prefer the `toRegex()` function but I'm not hung up on that)

Reply:
Agree with your suggested pattern and switched to `.toRegex()`. Separate commit below.
Does it bother you that I used triple-quotes over the escaped patterns you have elsewhere? (I prefer the reduced visual clutter when reading Regex).

I should probably try GitHub Actions but it hasn't been a priority since most of my source controlled work is on bitbucket. Not likely to happen tonight/soon.

Reply:
No strong opinion on triple-quotes for regex; what you have here works for me! No worries on the GitHub Actions, just a thought since this change is Windows specific. I can update the workflows some other time. (Or you can do a follow up PR if you want to attempt it)